### DISCUSSION TITLE
Cannot connect from external IP

### DISCUSSION THREAD

**Discussion by yanickrochon** [rustdesk/rustdesk#2191] [Q&A]
I have installed the server using the [installation script](https://rustdesk.com/docs/en/self-host/install/) and used the _external_ IP route. The ports 21115 ... 21119 (TCP) and 21116 (UDP) have been opened in the firewall to redirect all external requests to the server.

I can connect to the server via telnet through the _external_ IP (i.e. `telnet <public ip> 21116`) so the port forwarding works.

I also made a test on LAN, using two computers on the same local network, and they connect with no issue using the server's _internal_ IP address.

But trying to connect using the _external_ IP fails with a timeout.

In short:

1. Two computers on LAN can connect with no issue
2. Any computer trying to connect via the external IP times out
3. Connecting to the external IP using telnet works

What could be the issue?

**Comment by yanickrochon** [comment]
After some reading, this is apparantly a firewall problem. We are using Forgigate, so I'll have to ask for a LAN 2 LAN VIP port forwarding (or something).
